packages/app/src/searchFilters.tsx

```diff
@@ -127,10 +127,7 @@ export const useSearchPageFilterState = ({
   const [filters, setFilters] = React.useState<FilterState>({});
 
   React.useEffect(() => {
-    if (
-      !areFiltersEqual(filters, parsedQuery.filters) &&
-      Object.values(parsedQuery.filters).length > 0
```

Comment:
This check was preventing us from clearing the filter state if all filters were removed from the URL params (which happens when changing sources due to clicking on an attribute from a different source)